server/libs/userlibs.js

```diff
@@ -148,4 +149,100 @@ const logOut = function(request) {
     })
 }
 
-module.exports = {register, logIn, updateUser, logOut}
\ No newline at end of file
+const adminDeActivateUser = function(body) {
+    let id = body.uuid;
```

Comment:
use one flow in the code either do body.uuid in the where or use the local variable.
This should apply every where

Reply:
Done